tools/update_fake_backends.py

```diff
@@ -72,26 +285,45 @@ def _main():
             if name == "16":
                 name = "melbourne"
         if not args.backends or (name in args.backends or raw_name in args.backends):
-            if not os.path.isdir(os.path.join(args.dir, name)):
+            backend_dirname = os.path.join(args.dir, name)
+            if not os.path.isdir(backend_dirname):
                 print("Skipping, fake backend for %s does not exist yet" % name)
                 continue
+
+            sys.path.append(args.dir)
+            backend_mod = import_module(name)
+            fake_backend = getattr(backend_mod, "Fake" + name.capitalize())()
```

Comment:
I got this error for `ibm_oslo` which is related to what I commented in the [other PR](https://github.com/Qiskit/qiskit-terra/pull/8466/files#r1031381785).

```
  File "/Users/junye/Documents/GitHub/qiskit-dev/qiskit-terra/./tools/update_fake_backends.py", line 300, in _main
    summary = _Summary(props, fake_backend.properties())
AttributeError: 'FakeOslo' object has no attribute 'properties'
```

Here you assume that `Fake{Backend}` is a V1 backend and it has `properties` attribute. But `FakeOslo` is actually a V2 backend.

Reply:
This script should probably be updated for V2 backend

Reply:
Let's leave that for another PR, since it'll be a bit more complicated, most likely.  Right now, we can just skip the backend if the one we're given is V2.